castai/add-castai-removal-disabled/add-castai-removal-disabled.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: add-castai-removal-disabled
+  annotations:
+    policies.kyverno.io/title: Add CAST AI Removal Disabled
+    policies.kyverno.io/category: CAST AI, EKS Best Practices
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    kyverno.io/kyverno-version: 1.7.1
```

Comment:
This annotation means the version of Kyverno on which it was developed and confirmed to work. This should, at present, be at least 1.9 so please test and verify, then increment the value.

Reply:
Makes sense, updated and tested with 1.9.